### PR TITLE
Add manifest for org.kde.kruler

### PR DESCRIPTION
**Pull request by flyingcakes85**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by flyingcakes85** [comment]
bot, build org.kde.kruler

**Comment by flyingcakes85** [comment]
bot, build org.kde.kruler

File: org.kde.kruler.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "id": "org.kde.kruler",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kruler",
+    "rename-icon": "kruler",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland" ],
```
Comment (hfiguiere) [inline org.kde.kruler.json:9]:
```suggestion
    "finish-args": [
        "--share=ipc",
        "--socket=fallback-x11",
        "--socket=wayland"
    ],
```

File: org.kde.kruler.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            "name": "kruler",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                { "type": "git",
+                  "commit": "0077a55ea0cb8acfc5b02bd072d41f458546c5e9",
+                  "url": "https://invent.kde.org/kde/kruler.git" },
+                { "type": "patch",
+                  "paths": [
+                      "appdata.patch"
+                    ]
```
Comment (hfiguiere) [inline org.kde.kruler.json:22]:
```suggestion
                  "path": "appdata.patch"
```

File: org.kde.kruler.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+    "id": "org.kde.kruler",
+    "branch": "master",
```
Comment (hfiguiere) [inline org.kde.kruler.json:3]:
```suggestion
    "branch": "master",
```

**Comment by flyingcakes85** [comment]
bot, build org.kde.kruler

File: org.kde.kruler.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+    "id": "org.kde.kruler",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kruler",
+    "rename-icon": "kruler",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
```
Comment (hfiguiere) [inline org.kde.kruler.json:10]:
```suggestion
        "--socket=fallback-x11",
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]
still some changes

**Comment by flyingcakes85** [comment]
Pardon, I missed it while making changes. I've pushed a commit to fix it.

**Comment by flyingcakes85** [comment]
bot, build org.kde.kruler

**Comment by nedrichards** [comment]
@flathub/kde FYI

File: org.kde.kruler.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        "--socket=wayland"
+    ],
+
+    "modules": [
+        {
+            "name": "kruler",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                { "type": "git",
+                  "commit": "0077a55ea0cb8acfc5b02bd072d41f458546c5e9",
```
Comment (tsdgeos) [inline org.kde.kruler.json:20]: You need to use a tarball not a git commit

**Comment by flyingcakes85** [comment]
bot, build org.kde.kruler

File: org.kde.kruler.json (hunk 5 above)
Comment (flyingcakes85) [reply org.kde.kruler.json]: https://github.com/flathub/flathub/pull/2750/commits/443bfbb1d0f2ba5ef85ff9968a9b93981ec94aa7

File: org.kde.kruler.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+
+    "modules": [
+        {
+            "name": "kruler",
+            "buildsystem": "cmake-ninja",
+            "sources": [
```
Comment (barthalion) [inline org.kde.kruler.json:18]: The formatting is a little bit off here.

**Comment by barthalion** [comment]
All good from your perspective @tsdgeos?

File: org.kde.kruler.json (hunk 6 above)
Comment (flyingcakes85) [reply org.kde.kruler.json:18]: Lines 15-18 look okay to me; if there's something specific you'd like to suggest, please do.

I did find a formatting issue in the lines below. I've updated them : https://github.com/flathub/flathub/pull/2750/commits/71b46d00f8fb5a2568f2ae04c9972df8ec6171c2

**Comment by flyingcakes85** [comment]
bot, build org.kde.kruler

**Comment by travier** [comment]
LGTM. Thanks!
Could you link to where you have submitted the appdata patch upstream?

**Comment by travier** [comment]
https://invent.kde.org/graphics/kruler/-/merge_requests/5 > found it here. LGTM.

**Comment by tsdgeos** [comment]
Looks good to me

**Pull request closed without merging**